tests/e2e/vc_reboot_volume_lifecycle.go

```diff
@@ -79,57 +84,56 @@ var _ bool = ginkgo.Describe("Verify volume life_cycle operations works fine aft
 		9. Delete PVC, PV and Storage Class
 	*/
 
-	ginkgo.It("[csi-supervisor] [csi-guest] verify volume operations on VC works fine after vc reboots", func() {
-		storagePolicyName := GetAndExpectStringEnvVar(envStoragePolicyNameForSharedDatastores)
-		scParameters := make(map[string]string)
+	ginkgo.It("[csi-block-vanilla] [csi-supervisor] [csi-guest] verify volume operations on VC works fine after vc reboots", func() {
 
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		var storageclass *storagev1.StorageClass
+		var pvclaim *v1.PersistentVolumeClaim
+		var svcPVCName string
+		var err error
+		ginkgo.By("Creating Storage Class and PVC")
 		// decide which test setup is available to run
-		if supervisorCluster {
+		if vanillaCluster {
+			ginkgo.By("CNS_TEST: Running for Vanilla setup")
+			scParameters[scParamFsType] = ext4FSType
+			storageclass, pvclaim, err = createPVCAndStorageClass(client, namespace, nil, scParameters, "", nil, "", false, "")
+		} else if supervisorCluster {
 			ginkgo.By("CNS_TEST: Running for WCP setup")
+			ginkgo.By(fmt.Sprintf("storagePolicyName: %s", storagePolicyName))
 			profileID := e2eVSphere.GetSpbmPolicyID(storagePolicyName)
 			scParameters[scParamStoragePolicyID] = profileID
 			// create resource quota
 			createResourceQuota(client, namespace, rqLimit, storagePolicyName)
+			storageclass, pvclaim, err = createPVCAndStorageClass(client, namespace, nil, scParameters, "", nil, "", false, "", storagePolicyName)
 		} else {
 			ginkgo.By("CNS_TEST: Running for GC setup")
 			scParameters[svStorageClassName] = storagePolicyName
-		}
-		var storageclass *storagev1.StorageClass
-		var pvclaim *v1.PersistentVolumeClaim
-		var svcPVCName string // PVC Name in the Supervisor Cluster
-		var err error
-		ctx, cancel := context.WithCancel(context.Background())
-		defer cancel()
-		// decide which test setup is available to run
-		if guestCluster {
-			ginkgo.By("CNS_TEST: Running for GC setup")
+			createResourceQuota(client, namespace, rqLimit, storagePolicyName)
 			storageclass, pvclaim, err = createPVCAndStorageClass(client, namespace, nil, scParameters, "", nil, "", false, "")
-		} else {
-			ginkgo.By("CNS_TEST: Running for WCP setup")
-			storageclass, pvclaim, err = createPVCAndStorageClass(client, namespace, nil, scParameters, "", nil, "", false, "", storagePolicyName)
 		}
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 		defer func() {
 			err := client.StorageV1().StorageClasses().Delete(ctx, storageclass.Name, *metav1.NewDeleteOptions(0))
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}()
-		defer func() {
```

Comment:
Why did you remove this? If a PVC fails to bind the test will exit there and will not cleanup the PVC

Reply:
It is not removed. Only line number has changed hence it is showing in red. 
Delete PVC is still there in defer method line 129

Reply:
what I meant was to move the defer method before `fpv.WaitForPVClaimBoundPhase()`

Reply:
done